### DISCUSSION TITLE
Satellite Planetum-1 is not available on Stellarium

### DISCUSSION THREAD

**Discussion by axapet** [Stellarium/stellarium#3536] [Q&A]
Hello, 
the Planetum-1 satellite (COSPAR: 2022-057G. Catalog number: 52738/99398. Nanosatellite category: CubeSat 1U.) 
is not available in your aplication Stellarium. Please fix it.

Thank you.

Yours sincerely, Jan Peterka, CZ.

**Comment by alex-w** [comment]
Sorry, but what is problem to add it by yourself via Satellites plugin?
